data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/KafkaSecurityConfigurer.java

```diff
@@ -83,6 +83,10 @@ public class KafkaSecurityConfigurer {
     private static final String REGISTRY_BASIC_AUTH_USER_INFO = "schema.registry.basic.auth.user.info";
 
     private static final int MAX_KAFKA_CLIENT_RETRIES = 360; // for one hour every 10 seconds
+    private static final String SSL_ENGINE_FACTORY_CLASS = "ssl.engine.factory.class";
+    private static final String CERTIFICATE_CONTENT = "certificateContent";
```

Comment:
Is this valid name? I don't seem to find it.

Reply:
It's internal property that be used in `CustomClientSslEngineFactory` class to get the certificate content to setup TrustManager.